File: metric_tool/test/resources/kotlin/lcam/LcamC.kt
package io.company.app.view.basket.kotlin.lcam

internal class LcamC() {
    private lateinit var myField: String

    constructor(myFieldConstructorValue: String) : this() {
        myField = myFieldConstructorValue
    }

    fun setField(value: String) {
        myField = value
    }

    fun getField(): String {
        return myField
    }
}

File: metric_tool/test/resources/dart/lcam/lcam_c.dart
library mylib;

class LcamC {
  LcamC();

  LcamC.secondaryConstructor(myField) {
    this._myField = myField;
  }

  String _myField = "";

  void setField(String myField) {
    _myField = myField;
  }

  String getField() {
    return _myField;
  }
}
